free-1.md

```diff
@@ -57,12 +82,18 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I think the time spent would be evenly split between figuring out how to make this into a comprehensive language, and doing user studies to figure out what's important to people that's also feasible to calculate outside of the game. The actual calculations the game uses are fairly well-known, so its pretty clear how to make the backend work. However, there would be a good amount of time spent inputting all of the data for the existing heroes, but it would be a one-time setup cost, not a thing to continue devloping over time. 
 
```

Comment:
Great point that the data is well known by the community.  Having to manually test things out in-game to get all the necessary data would be a pain.